products.go

```diff
@@ -7,12 +7,16 @@ type Products []Product
 
 // Product is an item that can be bought by a customer.
 type Product struct {
-	// ID is the identifier of the product.
-	ID int64
-	// CreatedAt is the date and time the product was created.
-	CreatedAt time.Time
 	// BodyHTML is a description of the product. Supports HTML formatting.
 	BodyHTML string
+	// CreatedAt is the date and time the product was created.
+	CreatedAt time.Time
+	// Handle is a unique human-friendly string for the product.
+	Handle string
+	// ID is the identifier of the product.
+	ID int64
```

Comment:
Not ID at the top, not sure what it's fixing. Might be nicer to have the ID at the top.

Reply:
doing these in the exact order they are in the docs to be consistent with them :)

Reply:
Ah fair enough.